backend/.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "python.linting.flake8Enabled": true,
+    "python.linting.enabled": true,
+    "python.linting.flake8Args": ["--max-line-length=150", "--ignore=F405,F403"],
+}
```

Comment:
i suppose i am fine with this... but for usual case most devs have their own settings json (just saying)

Reply:
Removed `settings.json`, I figured out how to get the linting working i just needed another block for formatting if anyone else is reading

```json
"python.linting.enabled": true,
  "python.linting.flake8Enabled": true,
  "python.formatting.autopep8Args": [
    "--max-line-length=150",
    "--ignore=F405,F403"
  ],
  "python.linting.flake8Args": [
    "--max-line-length=150",
    "--ignore=F401"
  ],
  ```
  in your global `settings.json`